Classes/Common/Helper.php

```diff
@@ -893,31 +855,23 @@ public static function whereClause($table, $showHidden = false)
      */
     public static function whereExpression($table, $showHidden = false)
     {
-        $expressionBuilder = GeneralUtility::makeInstance(ConnectionPool::class)
-            ->getQueryBuilderForTable($table)
-            ->expr();
-
         if (\TYPO3_MODE === 'FE') {
             // Should we ignore the record's hidden flag?
-            $ignoreHide = -1;
+            $ignoreHide = 0;
```

Comment:
Setting `$ignoreHide` to `-1` will nowadays lead to TYPO3 using the configured page default. Therefore we have to explicitly set the value to `0` or `1`.
See method `PageRepository->enableFields()`.